modules/exploits/windows/scada/delta_ia_commgr_bof.rb

```diff
@@ -0,0 +1,75 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Delta Electronics Delta Industrial Automation COMMGR 1.08 Stack Buffer Overflow',
+      'Description'    => %q{
+        This module exploits a stack based buffer overflow in Delta Electronics Delta Industrial
+        Automation COMMGR 1.08. The vulnerability exists in COMMGR.exe when handling specially
+        crafted packets. This module has been tested successfully on Delta Electronics Delta
+        Industrial Automation COMMGR 1.08 over
+          Windows XP SP3,
+          Windows 7 SP1, and
+          Windows 8.1.
+      },
+      'Author'         =>
+        [
+          't4rkd3vilz',    # PoC
+          'hubertwslin'    # Metasploit module
+        ],
+      'References'     =>
+        [
+          [ 'EDB', '44965'],
```

Comment:
```ruby
[ 'BID', '104529' ],
[ 'URL', 'https://ics-cert.us-cert.gov/advisories/ICSA-18-172-01' ],
[ 'ZDI', '18-586' ],
[ 'ZDI', '18-588' ]
```

Reply:
This was not actually resolved. See #10775, which is incomplete.